_ext/ytmusic.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Mopidy-YTMusic
+service: YouTube Music
+type: backend
+dev:
+  github: OzymandiasTheGreat/mopidy-ytmusic
+dist:
+  pypi: Mopidy-YTMusic
+logo: /media/ext/youtubemusic.svg
+images:
+- /media/ext/youtubemusic.svg
```

Comment:
```suggestion
```
`images` is for images showing the extension in use.

Reply:
Thanks, I was clearly not paying close enough attention.